ecosystem/resources/members/quantumcat.toml

```diff
@@ -48,117 +48,117 @@ passed = false
 test_type = "development"
 passed = false
 package = "qiskit-terra"
-package_version = "0.21.0"
-terra_version = "0.21.0"
+package_version = "0.37.0"
+qiskit_version = "0.37.0"
 timestamp = 1678827878.289239
 
 [[historical_test_results]]
 test_type = "stable"
 passed = false
 package = "qiskit-terra"
-package_version = "0.17.4"
-terra_version = "0.17.4"
+package_version = "0.27.0"
+qiskit_version = "0.27.0"
 timestamp = 1678827878.289244
 logs_link = "https://github.com/qiskit-community/ecosystem/actions/runs/2998938614"
 
 [[historical_test_results]]
 test_type = "development"
 passed = false
 package = "qiskit-terra"
-package_version = "0.22.0"
-terra_version = "0.22.0"
+package_version = "0.39.0"
+qiskit_version = "0.39.0"
 timestamp = 1678827878.289247
-logs_link = "https://github.com/qiskit-community/ecosystem/actions/runs/3180920000"
+logs_link = "https://github.com/qiskit-community/ecosystem/actions/runs/3180.36.00"
 package_commit_hash = "53e215c31cf3aea51a623dc22883ac92fe74d0b9"
 
 [[historical_test_results]]
 test_type = "stable"
 passed = false
 package = "qiskit-terra"
-package_version = "0.21.2"
-terra_version = "0.21.2"
+package_version = "0.38.0"
+qiskit_version = "0.38.0"
 timestamp = 1678827878.28925
 logs_link = "https://github.com/qiskit-community/ecosystem/actions/runs/3225638565"
 
 [[historical_test_results]]
 test_type = "stable"
 passed = false
 package = "qiskit-terra"
-package_version = "0.22.0"
-terra_version = "0.22.0"
+package_version = "0.39.0"
+qiskit_version = "0.39.0"
 timestamp = 1678827878.289252
 logs_link = "https://github.com/qiskit-community/ecosystem/actions/runs/3368224447"
 
 [[historical_test_results]]
 test_type = "stable"
 passed = false
 package = "qiskit-terra"
-package_version = "0.22.2"
-terra_version = "0.22.2"
+package_version = "0.39.2"
+qiskit_version = "0.39.2"
 timestamp = 1678827878.289254
 logs_link = "https://github.com/qiskit-community/ecosystem/actions/runs/3521951857"
 
 [[historical_test_results]]
 test_type = "development"
 passed = false
 package = "qiskit-terra"
-package_version = "0.23.0"
-terra_version = "0.23.0"
+package_version = "0.40.0"
+qiskit_version = "0.40.0"
 timestamp = 1678827878.289257
 logs_link = "https://github.com/qiskit-community/ecosystem/actions/runs/3937703088"
-package_commit_hash = "af0b2220b75fd5cf3980576b1a31f9c4ceb9f99f"
+package_commit_hash = "af0.39.0b75fd5cf3980576b1a31f9c4ceb9f99f"
```

Comment:
Do you know why this changed?

Reply:
It seems my way to do it translate the `b...` into a notation and match with my sed.